Makefile

```diff
@@ -14,7 +15,11 @@ debug:
 
 .PHONY: bin
 bin:
+ifeq ($(GOOS),windows)
```

Comment:
We don't need the `-o` at all. You can do just `go build ${BUILD_FLAGS} cmd/odo/odo.go` and it will produce the currently named binary

Reply:
Doesn't work without ```-o``` flag. Seems flag ```-o``` is essential to name a a binary. 
```
$ make bin
go build -ldflags="-w -X github.com/openshift/odo/pkg/odo/cli/version.GITCOMMIT=ffa2756f" odo cmd/odo/odo.go
can't load package: package odo: cannot find package "odo" in any of:
        C:\Go\src\odo (from $GOROOT)
        C:\Users\cdk\go\src\odo (from $GOPATH)
can't load package: package cmd/odo/odo.go: cannot find package "cmd/odo/odo.go" in any of:
        C:\Go\src\cmd\odo\odo.go (from $GOROOT)
        C:\Users\cdk\go\src\cmd\odo\odo.go (from $GOPATH)
make: *** [bin] Error 1
```

Reply:
not like this :-)
you still have `odo` there.
Command should be `go build ${BUILD_FLAGS} cmd/odo/odo.go` not `go build ${BUILD_FLAGS} odo cmd/odo/odo.go

Reply:
Ahh... Right

Thanks for the tip :)